x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/deployment/DeploymentManager.java

```diff
@@ -442,22 +460,32 @@ void startPriorityProcessWorker() {
             executorServiceForProcess.submit(priorityProcessWorker::start);
         }
 
-        synchronized void stopProcess() {
-            isStopped = true;
-            resultProcessor.stop();
-            stateStreamer.cancel();
+        synchronized void forcefullyStopProcess() {
```

Comment:
Just refactored the common pieces into functions that are shared between this method and `stopProcessAfterCompletingPendingWork`